### ISSUE TITLE
[Feature]: hide Splash screen option for offline use

### ISSUE THREAD

**Issue by danilw** [vladmandic/automatic#3078]
### Feature description

I had to use vladmandic/automatic in offline - and splash screen popup with error "500 can not connect" in 10 sec after web-UI web page refresh/load.

**Splash screen blocking web-UI in offline-no-internet** - when UI and SD is completely functional offline.

Had to edit `function createSplash()` https://github.com/vladmandic/automatic/blob/master/javascript/loader.js#L27 just return from it.

I saw https://github.com/vladmandic/automatic/issues/2006 - but Splash does block entire UI for no reason.

**Simple solution can be** - add button to "hide splash screen" on splash screen.

**Comment by vladmandic** [comment]
splash screen cannot be disabled in settings as it would defeat the point - once settings are available, that means server is almost ready and splash screen goes away. so how to make it configurable?

but i don't understand why are you getting errors in splash screen to start with? splash screen does not load anything from the internet, it loads img from the same url that initial page load started from, so if that is not available, what is?

can you describe the exact scenario and actual erorrs?

**Comment by danilw** [comment]
>can you describe the exact scenario and actual erorrs?

Had storm outside - so internet was not working.
Im under router-NAT my PC is static 192.168.1.100 IP from router.
Router connected to modem, and when no connection to provider - modem turned into 192.168.100.1 webpage with modem status.

Webbrowser load page/DNS requests like `ping google.com` just "stuck" till it drop by application timeout, but no response.

Options for `vladmandic/automatic` in `webui-user.sh`  - `--use-xformers --medvram --skip-all --use-cuda --no-half-controlnet`.

When I opened UI - I had splash screen - splash screen after UI loaded and everything was working, in 10-20 sec
**That splash screen show "error 500" on loading** `https://vladmandic.github.io/automatic/motd`
But same error was in server-terminal-output - so I found https://github.com/vladmandic/automatic/blob/master/modules/api/server.py#L19 

And after removing it - **no more error in server terminal** .... but splash still shown with same "error 500" in 10-20 sec after loading.

So as I said - I just removed https://github.com/vladmandic/automatic/blob/master/javascript/loader.js#L27 by return from it - and it worked.

I tested now - by disconnecting internet cable from PC - I did not got splash screen, but I tested after I connected to internet today - so maybe there some cache or something.

It was very annoying to see this splash screen pop-up after load, and offline debugging...
